### PR TITLE
Hide more section when everything is disabled and update tests

### PR DESCRIPTION
**Pull request by abelgardep**
## Related Issues
App: #3271 

Library PR (if needed):

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

**Comment by jabarros** [comment]
LGTM

**Review by jabarros — APPROVED** [review]

File: changelog/unreleased/3296 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Enhancement: Hide more section if all options are disabled
```
Comment (jesmrec) [inline changelog/unreleased/3296:1]: I'd do a small change here: `Hide "More" section in Settings ... `.  Quotation marks help to distinguish the literal.

File: changelog/unreleased/3296 [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+A blank view was shown when all options in more subsection were disabled. Now, the
```
Comment (jesmrec) [inline changelog/unreleased/3296:3]: same here

**Comment by jesmrec** [comment]
Checks done:

- [X] All subsections enabled -> `More` visible
- [X] All subsections disabled -> `More` not visible
- [X] Only one subsection enabled -> `More` visible

Approved from my side

**Pull request merged**